pallets/vc-management/src/schema.rs

```diff
@@ -17,12 +17,21 @@
 // VC Schema
 // According to https://w3c-ccg.github.io/vc-json-schemas/, it defines JSON Schema for W3C Verifiable Credential.
 
-use crate::{vc_context::Status, Config};
+use crate::Config;
 use codec::{Decode, Encode, MaxEncodedLen};
 use core_primitives::{SchemaContentString, SchemaIdString};
 use scale_info::TypeInfo;
 use sp_std::vec::Vec;
 
+#[derive(Encode, Decode, Clone, Debug, PartialEq, Eq, TypeInfo, MaxEncodedLen)]
+pub enum Status {
```

Comment:
just wondering why we need to keep this one

Reply:
It's used by `VCSchema` struct defined below.

Reply:
There were no plans of removing schema/schema's status so this enum cannot be deleted. You think we should remove it ?

Reply:
Ah, I guess I'm missing something. If it is not part of the vc-registry then, sure, it is out of the scope 👍